beacon_node/beacon_chain/src/validator_monitor.rs

```diff
@@ -347,12 +410,20 @@ impl<T: EthSpec> ValidatorMonitor<T> {
             if let Some(i) = monitored_validator.index {
                 monitored_validator.touch_epoch_summary(current_epoch);
 
+                let i = i as usize;
+
+                // Cache relevant validator info.
+                if let Some(balance) = state.balances().get(i) {
+                    monitored_validator.with_epoch_summary(current_epoch, |summary| {
+                        summary.register_validator_total_balance(*balance)
```

Comment:
It seems a bit funky that this bypasses `individual_tracking`, but I guess that's what we want. It doesn't contribute to metric bloat, so should be fine.